tracer/src/Datadog.Trace/AppSec/Waf/RCMCapabilitiesHelper.cs

```diff
@@ -17,8 +17,12 @@ internal static class RCMCapabilitiesHelper
 {
     private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(RCMCapabilitiesHelper));
 
-    private static readonly Dictionary<BigInteger, Version> _CapabilitiesVersion = new Dictionary<BigInteger, Version>()
+    private static readonly Dictionary<BigInteger, Version> _CapabilitiesByWafVersion = new Dictionary<BigInteger, Version>()
```

Comment:
nit: standard naming would make this `_capabilitiesByWafVersion`

Reply:
Thanks! Fixed!